### ISSUE TITLE
No 'version' key in the manifest file for custom integration 'thingy52'.

### ISSUE THREAD

**Issue by manuelachterberg** [wlgrd/thingy52_homeassistant#10]
Hi there,

I use your custom integration on a daily basis and now an update is required by Home Assistant:

"No 'version' key in the manifest file for custom integration 'thingy52'. As of Home Assistant 2021.6, this integration will no longer be loaded. Please report this to the maintainer of 'thingy52'"

**Comment by andreas-2017** [comment]
I had the same problem. Fixed it by adding a version line into /config/custom_components/thingy52/manifest.json:

```
{
  "domain": "thingy52",
  "name": "Thingy:52 Sensor for Home Assistant",
  "version": "0.0.2",
  "documentation": "https://github.com/wlgrd/thingy52_homeassistant",
  "dependencies": [],
  "codeowners": ["@wlgrd"],
  "requirements": ["bluepy"]
}
```

**Comment by wlgrd** [comment]
I'm not using this myself anymore, but I added @andreas-2017 fix to the manifest file in b2f644607b989d0ac79fc403f34c6b0af9dc4a7b
